client/src/components/Explorer/actions.js

```diff
@@ -68,7 +68,7 @@ export function toggleExplorer(id) {
       }
 
       // We need to get the title of the starting page, only if it is not the site's root.
-      const isNotRoot = id !== 1;
+      const isNotRoot = id !== 'root';
```

Comment:
If I'm not mistaken, the purpose of this line is to treat the _global_ tree root as a special case - if the explorer starts from there, we leave the menu heading at its default value ("Pages") rather than setting it to the root page's title ("Root"). Changing the test to `'root'` (i.e. the user's explorable root) breaks this in a subtle but important way: if a user's explorable root is further down the tree, they will no longer see that page title as the menu heading:

<img width="495" alt="Screenshot 2019-11-28 at 00 24 36" src="https://user-images.githubusercontent.com/85097/69768125-3ff4a700-1177-11ea-835b-21207f072f53.png">